data/json/items/ranged.json

```diff
@@ -4194,7 +4194,38 @@
     "reload": 800,
     "loudness" : 175,
     "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "bore", 1 ],[ "sights", 1 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "sights", 1 ]]
- },{
+ },
+ {
+    "id": "m134",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "M134D-H Minigun",
+    "description": "The M134 Minigun is a 7.62x51 mm NATO, six-barreled machine gun.  It features Gatling-style rotating barrels with an electric motor.  Its' recoil is too powerful for firing unmounted, but if you could attach it to a vehicle and find enough ammo for it, it would become a devastating weapon. ",
+    "price": 5500000,
+    "material": "hardsteel",
+    "flags": ["MODE_BURST", "NEVER_JAMS", "MOUNTED_GUN"],
+    "skill": "rifle",
+    "ups_charges": 1,
+    "ammo": "308",
+    "weight": 27051,
+    "volume": 32,
+    "bashing": 18,
+    "cutting": 0,
+    "to_hit": -4,
+    "ranged_damage": 0,
+    "dispersion": 120,
+    "sight_dispersion": 200,
+    "aim_speed" : 0,
+    "recoil": 55,
+    "durability": 9,
+    "burst": 100,
+    "clip_size": 500,
+    "reload": 10000,
+    "loudness" : 150,
+    "valid_mod_locations": [[ "accessories", 4 ],[ "grip", 1 ],[ "rail", 1 ],[ "mechanism", 4 ],[ "sights", 1 ]]
```

Comment:
Do these mods even make sense for a mounted gun?  For example, swapping the sight will not do anything.